src/sass/_profile_list.scss

```diff
@@ -0,0 +1,104 @@
+.profile-list {
+  .profile-list-header {
+    display: flex;
+    justify-content: space-between;
+    margin: $spv--x-small 0 $spv--large 0;
+
+    .search-box {
+      max-width: 20rem;
+      width: 100%;
+    }
+
+    .margin-right {
+      margin-right: $sph--large;
+    }
+
+    .profiles-disabled {
+      visibility: hidden;
```

Comment:
Can we not render those items, instead of hiding it with css?

Reply:
Not with the current implementation. Because with `space-between`, the search box would stick to the right edge.